.github/workflows/cli-tests.yaml

```diff
@@ -36,14 +31,27 @@ jobs:
     timeout-minutes: 10
     steps:
       - uses: actions/checkout@v3
-      - uses: actions/setup-go@v3
+      - uses: actions/setup-go@v3.5.0
         with:
           go-version-file: ./go.mod
-          cache: false # use golangci cache instead
+      - name: Get current time
+        uses: josStorer/get-current-time@v2.0.2
+        id: current-time
+      - name: Mount golangci-lint cache
+        uses: actions/cache@v3
+        with:
+          path: |
+            ~/.cache/golangci-lint
+            ~/.cache/go-build
+            ~/go/pkg
+          key: golangci-lint-cache-${{ runner.os }}-${{ steps.current-time.outputs.day }}
+          restore-keys: |
+            golangci-lint-cache-${{ runner.os }}-
       - name: golangci-lint
-        uses: golangci/golangci-lint-action@v3.2.0
+        uses: golangci/golangci-lint-action@v3.4.0
```

Comment:
Do you happen to know if the go action already handles these ones? Should the linter just be caching it's own directory?

Reply:
That's a good point, I think the answer is possibly yes. But the linter will cache for all code instead of just a single go.mod (in the event we build more than one project in the repo)

I think the savings of this change plus a follow up for doing the same for golang will be more than enough.